projects/ngrx.io/content/guide/eslint-plugin/rules/signal-store-should-prefer-protected-state.md

```diff
@@ -0,0 +1,42 @@
+# signal-store-should-prefer-protected-state
+
+A Signal Store should prefer protected state.
+
+- **Type**: problem
+- **Fixable**: Yes
+- **Suggestion**: No
+- **Requires type checking**: No
+- **Configurable**: No
+
+<!-- Everything above this generated, do not edit -->
+<!-- MANUAL-DOC:START -->
+
+## Rule Details
+
```

Comment:
I'd add there a little explanation as well.

```suggestion

This rule should ensure that state changes are only managed by the Signal Store to prevent unintended modifications and provide clear ownership of where changes occur.
```

Reply:
Thanks @rainerhahnekamp , applied